homeassistant/components/modbus/base_platform.py

```diff
@@ -161,6 +164,11 @@ def __init__(self, hub: ModbusHub, config: dict) -> None:
         self._scale = config[CONF_SCALE]
         self._offset = config[CONF_OFFSET]
         self._count = config[CONF_COUNT]
+        self._nan_value = None
```

Comment:
Why having 2 variables that are essential the same ?

Reply:
this is one should be resolved